service/setup.md

```diff
@@ -0,0 +1,14 @@
+## Pre-requisites
+## Configuration
+1. Environment Variabls
+    1. ekstep_telemetry_auth : authorization header value
+    2. ekstep_telemetry_api_base_url : telemetry service base url "https://qa.ekstep.in/api"
+
+## Build
+1. Run "mvn clean install" from "sunbird-mw/services" to build the services.

```

Comment:
To be updated. Reference to 'sunbird-mw'.

Reply:
Not yet done.

Reply:
Done